runtime-decompiler/src/main/java/org/jrd/backend/communication/PossibleTopLevelError.java

```diff
@@ -0,0 +1,24 @@
+package org.jrd.backend.communication;
+
+public class PossibleTopLevelError extends PossibleError {
```

Comment:
Same naming issue as its superclass.

Reply:
Not sure if it is worthy... We agred on ErrorCandidate, right? and TopLeverlErrorCandidate

Reply:
already fixed